Cargo.toml

```diff
@@ -14,3 +14,9 @@ exclude = [
 # commands like `cargo run` will run the CLI binary by default.
 # See: https://doc.rust-lang.org/cargo/reference/workspaces.html#package-selection
 default-members = [ "cli" ]
+
+[workspace.package]
+version = "0.4.2"
```

Comment:
yessss :heart:

Reply:
Unfortunately this completely broke the `make package-check` target which I was previously unaware of, so I've backed out these changes 😞

Reply:
:pensive: oh dear. right, because `publish.rs` is doing some ad-hoc toml parsing...

ah well. next time.